_includes/contact-us.html

```diff
@@ -2,7 +2,7 @@
     <h2>Contact <span class="purple-font">US</span></h2>
     <div class="row">
         <div class="col-md-6">
-            <h3>Looking to volunteer?</h3>
+            <h3><br>Looking to volunteer?</h3>
```

Comment:
I'm not sure the `<br />` tag is necessary.

Reply:
removed.